hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/handlers/NodeUpdateHandler.java

```diff
@@ -166,7 +175,7 @@ private Node.Builder updateNode(@NonNull final NodeUpdateTransactionBody op, @No
         if (op.hasDeclineReward()) {
             nodeBuilder.declineReward(op.declineReward());
         }
-        if (op.hasGrpcProxyEndpoint()) {
+        if (nodeConfig.webProxyEndpointsEnabled() && op.hasGrpcProxyEndpoint()) {
```

Comment:
Since we check the feature is enabled earlier, we don't need to check here again